app/app/src/androidTest/kotlin/com/hedvig/app/feature/loggedin/LoggedInScreen.kt

```diff
@@ -1,17 +1,17 @@
 package com.hedvig.app.feature.loggedin
 
+import com.hedvig.android.feature.chat.ui.ChatActivity
 import com.hedvig.app.R
-import com.hedvig.app.feature.chat.ui.ChatActivity
 import io.github.kakaocup.kakao.bottomnav.KBottomNavigationView
 import io.github.kakaocup.kakao.common.views.KView
 import io.github.kakaocup.kakao.intent.KIntent
 import io.github.kakaocup.kakao.screen.Screen
 
 class LoggedInScreen : Screen<LoggedInScreen>() {
-  val root = KView { withId(R.id.loggedInRoot) }
-  val bottomTabs = KBottomNavigationView { withId(R.id.bottomNavigation) }
+  val root = KView { withId(R.id.chat) }
+  val bottomTabs = KBottomNavigationView { withId(R.id.chat) }
 
-  val openChat = KView { withId(R.id.openChat) }
+  val openChat = KView { withId(R.id.chat) }
```

Comment:
This was broken so I just put whatever IDs to make it not complain. We wanna delete this anyway, but didn't do it since it may cascade more than it's worth right now.